### PR TITLE
fix: allow for same filenames to be added

### PR DESCRIPTION
**Pull request by daKmoR**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

fixes https://github.com/webpack-contrib/karma-webpack/issues/400

### Additional Info

adds a hash to the webpack entry list so same filenames can exist.

### CODE REVIEW

**Comment by daKmoR** [comment]
@matthieu-foucault hey, would you be so kind and release this? 🤗  pretty please 🤗

**Comment by matthieu-foucault** [comment]
Will do tomorrow :)

File: lib/karma-webpack.js [hunk 1]
```diff
@@ -65,7 +83,7 @@ function configToWebpackEntries(config) {
 
   const webpackEntries = {};
   filteredFiles.forEach((filePath) => {
-    webpackEntries[path.parse(filePath).name] = filePath;
+    webpackEntries[getPathKey(filePath)] = filePath;
```
Comment (matthieu-foucault) [inline lib/karma-webpack.js]: Why do we need this?
Can't we just use `filePath` as the attribute key?

Comment (daKmoR) [reply lib/karma-webpack.js]: this is the key that webpack uses internally... it does not like '/' in it...
also it is the url that get's displayed when you build... so if that is too long it's quite annoying :see_no_evil:

**Comment by stevenvachon** [comment]
Can we merge this, please?

**Comment by matthieu-foucault** [comment]
:heavy_check_mark: Published under `karma-webpack@dev`

**Pull request merged**